Orange/widgets/classify/owloadclassifier.py

```diff
@@ -97,9 +97,9 @@ def load(self, filename):
         try:
             classifier = pickle.load(open(filename, "rb"))
         except pickle.UnpicklingError:
-            raise  # TODO: error reporting
+            raise  pickle.UnpicklingError("Failed to load '{}'.".format(filename))
```

Comment:
Only one space should appear between `raise` and argument, here and below..
Please amend your commit.